app/views/comments/_help.html.haml

```diff
@@ -1,4 +1,5 @@
-#help{:class=>"collapse"}
+- unique_id = '' if unique_id == nil
```

Comment:
Why is this needed here?

Reply:
for project's form since theres no id in project form

Reply:
Yes, but if is `nil` it will be transformed into an empty string.

Reply:
i put this condition because of "undefined local variable or method `unique_id' " since I didnt defined unique_id
i will define it as the local variable while rendering comments/help (like done in comment)